third_party/2and3/pymysql/cursors.pyi

```diff
@@ -22,12 +22,24 @@ class Cursor:
     def executemany(self, query: str, args) -> int: ...
     def callproc(self, procname, args=...): ...
     def fetchone(self) -> Optional[Gen]: ...
-    def fetchmany(self, size: Optional[int] = ...) -> Optional[Gen]: ...
+    def fetchmany(self, size: Optional[int] = ...) -> Union[Optional[Gen], List[Gen]]: ...
     def fetchall(self) -> Optional[Tuple[Gen, ...]]: ...
     def scroll(self, value, mode=''): ...
     def __iter__(self): ...
 
 class DictCursor(Cursor):
     def fetchone(self) -> Optional[Dict[str, Any]]: ...
-    def fetchmany(self, size=None) -> Optional[Tuple[Dict[str, Any], ...]]: ...
+    def fetchmany(self, size: Optional[int] = ...) -> Optional[Tuple[Dict[str, Any], ...]]: ...
     def fetchall(self) -> Optional[Tuple[Dict[str, Any], ...]]: ...
+
+class DictCursorMixin:
+    dict_type = ...  # type: dict
```

Comment:
Don't have time to check right now, but should this be `Type[Dict]`?

Also, it's preferred to use PEP 526 syntax (`dict_type: Type[Dict] = ...`).

Reply:
I'm changing this to `# type: Any` for now. From the pymysql code it needs something dict like, but not sure what that means specifically for types.